frontend/pages/manage/roles/index.tsx

```diff
@@ -46,14 +46,14 @@ const ViewRole = ({ role, refetch }: { role: RolesType; refetch: () => void }):
     const [showModal, setShowModal] = useState(false)
 
     return (
-        <div>
+        <>
             <Button
                 usage="toggle-modal"
                 onClick={() => {
                     setShowModal(true)
                 }}
             >
-                <Icons name="eye" />
+                <Icons name="eye" size="18" />
```

Comment:
I think the design called for size 20

Reply:
I consulted with Sir Jason regarding this adjustment, since icon size 20 looked too big compared to the text size.